src/TestExplorer.ts

```diff
@@ -0,0 +1,153 @@
+import * as ko from "knockout";
```

Comment:
Can you rework this file without knockout? Would strongly prefer to avoid adding new knockout code.